depends/common/chailove/CMakeLists.txt

```diff
@@ -3,6 +3,15 @@ project(chailove)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR})
 
+include(FetchContent)
+FetchContent_Declare(
+    libretrochailove
+    GIT_REPOSITORY https://github.com/libretro/libretro-chailove.git
+    GIT_TAG 6cf08ec1c0a441f2ebeb942ec0a28d9037b83816
+    SOURCE_DIR chailove
```

Comment:
We'll want to change `SOURCE_DIR` to where we want the files to be downloaded to.